src/datafeed.js

```diff
@@ -10,34 +10,34 @@ import {
 
 const lastBarsCache = new Map();
 
+// DatafeedConfiguration implementation
 const configurationData = {
+	// Represents which interval choices for bars your datafeed supports
```

Comment:
https://git.xtools.tv/tv/tradingview/-/merge_requests/39175#note_1968843
```suggestion
	// Represents the resolutions for bars supported by your datafeed
```